google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -187,6 +199,7 @@ protected void applyEditorTo(AppEngineDeploymentConfiguration configuration)
     configuration.setDockerFilePath(dockerFilePathField.getText());
     configuration.setAppYamlPath(appYamlPathField.getText());
     configuration.setConfigType(getConfigType());
+    configuration.setVersion(versionOverrideCheckBox.isSelected() ? versionIdField.getText() : null);
 
```

Comment:
yes, I'll add a regex. The pattern is printed out from gcloud when an invalid version string is sent.